cnc_ctrl_v1/Report.cpp

```diff
@@ -236,7 +236,20 @@ void  returnPoz(){
     static unsigned long lastRan = millis();
     
     if (millis() - lastRan > POSITIONTIMEOUT){
+        char tmpbuf[80], buflen=0;
+        memset(tmpbuf, 0, sizeof(tmpbuf));
         
+        for (int i=0;i<3;i++){
+            if(axes[i].stalled > 5){
+                buflen+=snprintf(tmpbuf+buflen, sizeof(tmpbuf)-buflen, "%c %s ", axes[i].name(), (axes[i].moved > 5) ? F("STALL") : F("UNHK "));
```

Comment:
This is nice and compact but hard to read.  I would be inclined to waste program space, which we will have a good amount of, to make this easier for humans to comprehend.

Also, it doesn't seem to me that moved has the value that you are testing for here.  I think you want moved to have the value of delta maybe?

Reply:
Yeah, I goofed.  Moved is a boolean.  If it's ever moved, it's stalled.  Otherwise, it's unhooked.